Python/Python_news_reader/Python news reader.py

```diff
@@ -0,0 +1,50 @@
+'''
+This python program collects the top news headlines and their description from 'newsapi.org' website and will read the news headlines and
+description in his own voice.
+News API is a simple and easy-to-use API that returns JSON metadata for headlines and articles live all over the web right now.
+Visit newsapi.org for documentation and for your API key
+'''
+
+import time
+from win32com.client import Dispatch
+import requests
+
+file = open("Url.txt", 'r+')
```

Comment:
As a Best Practice, you don't need a File mentioning the link. Instead, you can create a `.env` and use something like `python-dotenv` to fetch the API Keys. Keep the `.env` in the Repository so that you don't accidentally push the API Keys. Keep a `.env.example` in the main repo so that the Users can know how to use it. Get the API Key from the `.env` file and use it by concatenating it to the URL.

Reply:
Done!